### DISCUSSION TITLE
Trying to get english speaking dataset to speak french

### DISCUSSION THREAD

**Discussion by MastracciN** [coqui-ai/TTS#1733] [General Q&A]
Hey guys, I've been struggling to train the LJSpeech dataset so that it can speak another language properly. The Coqui website makes it seem like it is possible to do through the program but I feel lost with training in general. I've been reading the docs on training and looking over the recipes available but still feel lost. I'm sorry for the vague "question" in general but any guidance would be greatly appreciated. Thank you so much

**Comment by p0p4k** [comment]
It is hard to help you if you do not provide details like what you tried so far, which model you chose, what kind of dataset you are using, what errors you encountered, etc. When you know your question is vague, elaborating it would be a good step forward.

**Reply by MastracciN** [reply]
Fair enough! I was hoping to use the LJSpeech dataset and wanted to get it speaking another language, probably french. I'm extremely uncertain about what model to use. Some of the examples have been in GlowTTS so I've been looking at that. Suggestions on which would be a good place to start would be welcome. I was looking at the recipe "train_vits_tts.py" under the multilingual folder. Reading through it, I am unsure if I am looking at the correct python file to achieve what I want. I was trying to find where I would declare in the code what dataset to use or how to setup the intention of speaking a language different than what the dataset is. I've seen under VitsConfig there is a phoneme_language variable, would I change that to fr to achieve proper french pronunciation? I'm also confused about the difference between a tts model and a vocoder model.

**Reply by p0p4k** [reply]
First, do you have French dataset? Audio + text transcripts?

**Reply by MastracciN** [reply]
Was hoping to use the css10 french dataset https://www.kaggle.com/datasets/bryanpark/french-single-speaker-speech-dataset

**Reply by p0p4k** [reply]
Now, if you just wanna train to generate only french language, arrange your dataset in LJSpeech format. Refer to the code corresponding to model of your choice in [here](https://github.com/coqui-ai/TTS/tree/dev/recipes/ljspeech). And read the documentation as well. You can even read the [notebooks](https://github.com/coqui-ai/TTS/tree/dev/notebooks) to get a quick understanding of the training process. Take your time, if stuck search for similar questions in earlier discussions/issues and if you are still stuck, post back here. Good luck!

**Reply by MastracciN** [reply]
I think I've misunderstood something. If I train the model on french speech, can I pass it an english speaker's dataset to generate audio on what that speaker would sound like in french?

**Reply by p0p4k** [reply]
Maybe you want to do voice conversion. Try using VITS model.
